OpenRA.Game/Support/ExceptionHandler.cs

```diff
@@ -24,25 +24,29 @@ public static void HandleFatalError(Exception ex)
 			Log.AddChannel("exception", exceptionName);
 
 			if (Game.EngineVersion != null)
-				Log.Write("exception", "OpenRA engine version {0}", Game.EngineVersion);
+				Log.Write("exception", $"OpenRA engine version {Game.EngineVersion}");
+
+			if (Game.Settings != null && Game.Settings.Player != null && Game.Settings.Player.Language != null)
+				Log.Write("exception", $"OpenRA Language: {Game.Settings.Player.Language}");
 
 			if (Game.ModData != null)
 			{
 				var mod = Game.ModData.Manifest.Metadata;
-				Log.Write("exception", "{0} mod version {1}", mod.Title, mod.Version);
+				Log.Write("exception", $"{mod.Title} mod version ${mod.Version}");
 			}
 
 			if (Game.OrderManager != null && Game.OrderManager.World != null && Game.OrderManager.World.Map != null)
 			{
 				var map = Game.OrderManager.World.Map;
-				Log.Write("exception", "on map {0} ({1} by {2}).", map.Uid, map.Title, map.Author);
+				Log.Write("exception", $"on map {map.Uid} ({map.Title} by {map.Author}).");
 			}
 
-			Log.Write("exception", "Date: {0:u}", DateTime.UtcNow);
-			Log.Write("exception", "Operating System: {0} ({1})", Platform.CurrentPlatform, Environment.OSVersion);
-			Log.Write("exception", "Runtime Version: {0}", Platform.RuntimeVersion);
+			Log.Write("exception", $"Date: {DateTime.UtcNow:u}");
+			Log.Write("exception", $"Operating System: {Platform.CurrentPlatform} ({Environment.OSVersion})");
+			Log.Write("exception", $"Runtime Version: {Platform.RuntimeVersion}", Platform.RuntimeVersion);
+			Log.Write("exception", $"System Language: {CultureInfo.InstalledUICulture} (Installed) {CultureInfo.CurrentCulture} (Current) {CultureInfo.CurrentUICulture} (Current UI)");
```

Comment:
What is this line for? None of the variables show up for me

Reply:
`de_DE` for all variables? On my end I only see the words in the brackets what confused me. Shouldn't there be some defaults?

Reply:
Yes, for all. I don't know why it doesn't work on Mac.

Reply:
It appears it is reading it as iv (Invariant Country)

Reply:
What locale is your system set to?

Reply:
It's UK english, using US keyboard. I tried to change it to other languages and it still didn't pick up

Reply:
Are you running @mono? https://github.com/dotnet/runtime/issues/27159

Reply:
For this PR I think adding a default value is enough.

Reply:
Is the value null on Mac?

Reply:
What would be the default?